vendor/VREffect.js

```diff
@@ -232,7 +232,7 @@ THREE.VREffect = function ( renderer, onError ) {
 	var cameraR = new THREE.PerspectiveCamera();
 	cameraR.layers.enable( 2 );
 
-	this.render = function ( scene, camera ) {
+	this.render = function ( scene, camera, renderTarget, forceClear ) {

```

Comment:
The vendor directory must not have any patched code. It's messy to maintain and easy to forget that we have ad-hoc changes. This PR assumes new functionality on the VR Effect and Three.js might not agree on the approach. This would force us to redo this patch or maintain a fork of VREffect. Let's share the changes on a PR on three.js to get their feedback before merging this.